lib/timescaledb/stats.rb

```diff
@@ -0,0 +1,22 @@
+require_relative './stats/continuous_aggregates'
+require_relative './stats/hypertables'
+require_relative './stats/job_stats'
+
+module Timescaledb
+  class Stats
+    # @param [Array<OpenStruct>] hypertables The list of hypertables.
+    # @param [Timescaledb:Connection] connection The PG connection.
+    def initialize(hypertables = [], connection = Timescaledb.connection)
```

Comment:
Can we use keyword arguments and have some default shortcut for all hypertables.

Like:

```suggestion
    def initialize(hypertables: :all, connection: Timescaledb.connection)
```


```T

Reply:
I also think we could match different types of arguments in the future like `hypertables: ["hypertable_name ~ 'metrics_.*'"]`

I'd love if we could accept `:all`, `:none`, and the rest of the options will be considered a where clause for the hypertables scope.

One remarkable thought is that the same patterns would also be very welcome in the continuous aggregates stats. So, I see it filtering more in a linux style and injecting some matching for the hypertable and caggs names.